kb/plane_geometry/formal_statements/statements/statements_scs/stat_straight_line_parallelism.scs

```diff
@@ -0,0 +1,14 @@
+stat_straight_line_parallelism <- statement;;
+
+stat_straight_line_parallelism -> rrel_key_sc_element: concept_straight_line; nrel_parallelism;;
+
+stat_straight_line_parallelism => nrel_main_idtf: [Акс.(прямая;параллельность*)] (*<- lang_ru;;*);;
+stat_straight_line_parallelism => nrel_main_idtf: [Ax.(straight line;parallelism*)] (*<- lang_en;;*);;
+
+stat_straight_line_parallelism <= nrel_sc_text_translation: 
+		... 
+		(* 
+		-> rrel_example: 
+			[Если две различные прямые параллельны третьей прямой, то они параллельны.] 

```

Comment:
It could be a sign of parallel lines. Please check it. If I am right, add according identifier